.readthedocs.yml

```
@@ -0,0 +1,25 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Remove report submodule from docs build
+submodules:
+  exclude: all
+
+
+# Build documentation in the docs/ directory with Sphinx
+sphinx:
+   configuration: docs/conf.py
+
+# Optionally build your docs in additional formats such as PDF
+formats:
+   - pdf
+
+# Optionally set the version of Python and requirements required to build your docs
+python:
+   version: 3.8
+   install:
+   - requirements: requirements/dev-requirements.txt
```

```suggestion
   - requirements: requirements/requirements.txt
```

Would update this to the runtime dependencies rather than the requirements for developers

---

Good catch! This should actually be doc-requirements

---

Done in ca34f1c